internal/manifest/oci_index_test.go

```diff
@@ -45,3 +49,113 @@ func TestOCI1IndexFromManifest(t *testing.T) {
 	// Extra fields are rejected
 	testValidManifestWithExtraFieldsIsRejected(t, parser, validManifest, []string{"config", "fsLayers", "history", "layers"})
 }
+
+func TestOCI1IndexChooseInstanceByCompressionAndListInternalFromBlob(t *testing.T) {
```

Comment:
`TestOCI1IndexChooseInstanceByCompression`, please; it’s a unit test targeting that method.